pkg/templates/charts/toggle/console-mce/templates/console-plugin.yaml

```diff
@@ -1,22 +1,28 @@
 # Copyright Contributors to the Open Cluster Management project
-apiVersion: console.openshift.io/v1alpha1
+apiVersion: console.openshift.io/v1
 kind: ConsolePlugin
 metadata:
-  name: mce
   annotations:
-    console.openshift.io/use-i18n: "true"
+    installer.multicluster.openshift.io/release-version: 2.7.0
+  labels:
+    backplaneconfig.name: multiclusterengine
+  name: mce
 spec:
+  backend:
+    service:
+      basePath: /plugin/
+      name: console-mce-console
+      namespace: { { .Values.global.namespace } }
```

Comment:
Did some sort of auto-formatter add the spaces between the braces? That may not be valid.